charts/kubefed/charts/controllermanager/crds/crds.yaml

```diff
@@ -457,6 +457,8 @@ spec:
     kind: KubeFedCluster
     listKind: KubeFedClusterList
     plural: kubefedclusters
+    shortNames:
+    - kfc
```

Comment:
I'd prefer this PR to focus on the single goal of adding K8s version to the CRD's status.

Reply:
Just found the `kubefedcluster` is a little verbose in everyday use lol. I prefer to keep it but if you don't like it I can remove it or seperate to another PR, no big deal.